Implement Card-Based UI for TV

atanuroy22 opened the pull request:
New Card UI added for TV 
Similar like hotstar type navigation
By default the default layout will be shown.
On enabling card ui will be enable 
On disabling card ui will be disable and return to default ui
Old Fetching logics are reused only the ui added
# Test apk
[jio-tv-server-app-b8.zip](https://github.com/user-attachments/files/22926053/jio-tv-server-app-b8.zip)
# Demo Video
https://github.com/user-attachments/assets/94fae0f7-3e4d-43d8-af1c-fde1d39c83d7

atanuroy22 committed:
autoplay fixed

Sometime autoplay not works when server not start automatically,so made a 1 time play 1st channel on enter the zone ui.so if i go to setting and come  back to zone ui ,then it play 1st channel again for 1 time.

atanuroy22 committed:
Merge branch 'develop' into test

atanuroy22 committed:
Fixed Enhanced autoplay and watchdog

Fixed Enhanced Autoplay that forcefully load backend fetched data to ui and play,And watchdog checks for 2s that any channel playing or not if playing then break or if not then play once and break

atanuroy22 committed:
Focus issue Fixed

In MultiSelectDropdown focus issue  occur when changing to  another checkbox ,that fixed

atanuroy22 committed:
Merge branch 'JioTV-Go:develop' into test

atanuroy22 committed:
Reverted Resize options

atanuroy22 committed:
Merge branch 'develop' into test

atanuroy22 committed:
New Card ui in Zoneui

Added disney style ui that can be changable from the tvscreenmenu

atanuroy22 committed:
Merge branch 'develop' into test

atanuroy22 committed:
selection fix on android

atanuroy22 committed:
Merge branch 'test' into test-card-ui

atanuroy22 committed:
Navigation fix for both tv and android

Card Ui fixed in m3u and jiotv for TV and android

atanuroy22 committed:
fix some scroll issue

atanuroy22 committed:
Merge branch 'JioTV-Go:develop' into test

atanuroy22 committed:
Merge branch 'JioTV-Go:develop' into test-card-ui

atanuroy22 committed:
Merge pull request #1 from atanuroy22/test-card-ui

Test card UI

atanuroy22 committed:
some issue fix

atanuroy22 committed:
Fixed Experimental toggle and Card color

Added experimental toggle and fixed channel card background Color

atanuroy22 committed:
Merge branch 'JioTV-Go:develop' into test

atanuroy22 committed:
Merge branch 'JioTV-Go:develop' into test

atanuroy22 committed:
Merge branch 'develop' into test

atanuroy22 committed:
Build error fix of channel selection

siddharthsky committed:
Refactor TV card UI to modular CardGrid components

- Changed card size as it was too large and only a few channels were visible on screen
- Added a demo EPG to the main card UI on focus (actual EPG data will be added later)
- Replaced `CardChannelLayout` and `CardChannelLayoutM3U` with new modular `CardGrid` and `CardGridM3U` components
- Improve code organization and maintainability for TV card UIs

siddharthsky commented:
Card UI has been refactored. The remaining task is to pass the EPG data into the card grid. Can you test and check if everything is working so we can move forward

siddharthsky committed:
Integrate EPG data into channel cards and refactor focus logic

- Improving navigation consistency
- Removed scroll animation
- Added EPG data to channel card

atanuroy22 committed:
Merge branch 'develop' into test

siddharthsky committed:
Merge branch 'develop' into test

atanuroy22 committed:
Card ui Focus issu fix

On last channel focus moving to setting ,that issue fixed

atanuroy22 commented:
Bro check the last pushed focus selector code, From last channel sometime selector moves to setting fixed. that works for me.
You check if it works fine then proceed otherwise i will revert back last push.
